tests/tests.js

```diff
@@ -145,7 +145,68 @@ QUnit.test("generateMonthObjects calendar method test", function(assert) {
 });
 
 
+// Testing checkit object 
+QUnit.module( "CheckIt Tests", {
+
+beforeEach: function() {
+    // Prepare something once for all tests
+    this.params = {startDate: "2017-02-14" , endDate: "2017-02-20" , calendarTitle: "Test Calendar"};
+    this.state = emptyCalendarState(this.params);
+    this.calendar = new Calendar(this.state);
+    this.checkit = new CheckIt('localStorage');
+}
+
+});
+
+function selectorNameTest(testName, selectorName, expected) {
+    QUnit.test(testName, function(assert) {
+        assert.expect(1);
+        assert.equal(selectorName.selector, expected);
```

Comment:
This is good to check, but it doesn't protect you against something changing on the page. For example, suppose you refactor how the calendar works and need to change your date form selectors. This test will still pass but the selector won't *work*.

Is there a good way to test that each of the selectors actually selects an element?